src/engine/training.ts

```diff
@@ -839,7 +839,7 @@ export class Model extends Container implements tfc.InferenceModel {
    * @returns Loss and metric values as an Array of `Scalar` objects.
    */
   /**
-   * @doc {heading: 'Models', subheading: 'Classes', configParamIndices: [2]}
+   * @doc {heading: 'Models', subheading: 'Classes', configParamIndices: [1]}
    */
```

Comment:
I think you can fit the whole thing on a single line, for conciseness, i.e., 

`/** @doc {heading: 'Models', subheading: 'Classes', configParamIndices: [1]} */`

Reply:
Unfortunately no, it has 81 characters if fitting on a single line.